templates/kernel/real-time/contact-us.html

```diff
@@ -19,71 +23,85 @@ <h1>Run real-time Ubuntu in production</h1>
-                  <label class="p-radio">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-industrial"
-                           name="your-industry-is"
-                           value="Industrial" />
-                    <span class="p-radio__label" id="your-industry-is-industrial">Industrial</span>
-                  </label>
-                  <label class="p-radio">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-automotive"
-                           name="your-industry-is"
-                           value="Automotive" />
-                    <span class="p-radio__label" id="your-industry-is-automotive">Automotive</span>
-                  </label>
-                  <label class="p-radio">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-telco"
-                           name="your-industry-is"
-                           value="Telco" />
-                    <span class="p-radio__label" id="your-industry-is-telco">Telco</span>
-                  </label>
-                  <label class="p-radio u-no-margin--bottom">
-                    <input class="p-radio__input"
-                           type="radio"
-                           aria-labelledby="your-industry-is-public-sector"
-                           name="your-industry-is"
-                           value="Public sector" />
-                    <span class="p-radio__label" id="your-industry-is-public-sector">Public sector</span>
-                  </label>
-                  <div class="js-other-container">
-                    <label class="p-radio">
-                      <input class="p-radio__input js-other-container__other-toggle"
-                             type="radio"
-                             aria-labelledby="your-industry-is-other"
-                             name="your-industry-is"
-                             value="other" />
-                      <span class="p-radio__label" id="your-industry-is-other">Other</span>
-                    </label>
-                    <input type="text"
-                           id="your-industry-is-other-specified"
-                           class="js-other-container__input u-hide"
-                           name="your-industry-is"
-                           placeholder="Please specify"
-                           style="margin-top: .25rem"
-                           aria-label="Other industry specified" />
-                  </div>
-                </div>
+    <form action="/marketo/submit"
+          method="post"
+          id="mktoForm_4914"
+          onsubmit="getCustomFields(event)">
+
+      <div class="p-section js-formfield">
```

Comment:
I have wrapped each fieldset in a 'js-formfield' class. We should always try to use a specific 'js-*' class when accessing HTML to avoid future breakage from class/id changes.